scripts/neighbor_advertiser

```diff
@@ -361,6 +361,7 @@ def wrapped_ferret_request(request_slice, https_endpoint, http_endpoint):
         response = requests.post(http_endpoint,
                                  json=request_slice,
                                  timeout=DEFAULT_REQUEST_TIMEOUT)
+        response.raise_for_status()
```

Comment:
I think this change makes sense, but it makes this part of `post_neighbor_advertiser_slice` obsolete since an exception will always be thrown if the response does not have one of the 2xx codes:
```
# Handle response errors
        if not response:
            log_error('Failed to set up neighbor advertiser slice, vip: {}, no response obtained'.format(ferret_service_vip))
            return None
        if response and not response.ok:
            log_error('Failed to set up neighbor advertiser slice, vip: {}, error_code: {}, error_content: {}'.format(ferret_service_vip, response.status_code, response.content))
            return None
```

I'd say you should probably delete this from `post_neighbor_advertiser_slice` since it's no longer going to be used. We could also incorporate some of the logging in the wrapper method if we want more specific debugging information. Let me know what you think.